com.unity.shadergraph/Editor/Drawing/Inspector/PropertyDrawerUtils.cs

```diff
@@ -25,12 +25,20 @@ public static Label CreateLabel(string text, int indentLevel = 0)
             return labelVisualElement;
         }
 
+        internal enum UIPrecisionForShaderGraphs
```

Comment:
This is interesting, any particular reason to do this as opposed to just directly binding to the Precision enum?

Reply:
Ah okay for naming purposes, I'd assume, the last entry is different which is a neat way to wrap around another enum

Reply:
Yes, this is purely a simple way to control the names that show up in the UI.